docs/articles/documentation/using-testcafe/programming-interface/index.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: docs
+title: TestCafe Programming Interface
+permalink: /documentation/using-testcafe/programming-interface/
+---
+# TestCafe Programming Interface
+
+This section provides reference for the API that allows you use TestCafe from your NodeJS applications.
+
+#### Point of Entry
+
+* the [createTestCafe](/testcafe/documentation/using-testcafe/programming-interface/module-level-members/#createTestCafe) factory function
+
+#### Contents
+
+* [TestCafe Class](/testcafe/documentation/using-testcafe/programming-interface/TestCafe/)
+* [Runner Class](/testcafe/documentation/using-testcafe/programming-interface/Runner/)
+* [BrowserConnection Class](/testcafe/documentation/using-testcafe/programming-interface/BrowserConnection/)
+* [Module-Level Members](/testcafe/documentation/using-testcafe/programming-interface/module-level-members/)
```

Comment:
I don't think we should have it here, since we already have a reference to `createTestCafe` above